aws-analytics-pinpoint/build.gradle

```diff
@@ -16,6 +16,7 @@
 apply plugin: 'com.android.library'
 apply from: rootProject.file("configuration/checkstyle.gradle")
 apply from: rootProject.file("configuration/publishing.gradle")
+apply plugin: 'kotlin-android'
```

Comment:
Ooh....Kotlin!  Before we commit to adding Kotlin to the library (no pun intended 😆 ) - 
 - Does adding this plugin have any impact on library build/compile time?
 - Does adding this plugin have any impact on library aar size?
 - Will this have any impact on customers, who are only using Java in their application?